src/components/ClassInfo.vue

```diff
@@ -236,7 +235,7 @@ export default {
     },
     clickRelatedSubject: function (subject) {
       this.$emit('push-stack', subject.id);
-      $('#cardBody').animate({ scrollTop: 0 });
+      document.getElementById('cardBody').scrollTop = 0;
```

Comment:
This no longer animates the scroll nicely.  Not sure if there is a way to animate this without jquery?  The scroll-behavior:smooth doesn't seem to be having the same effect.

Reply:
There are two ways we can do this: figure out the `CSS` animation or drop in a replacement for `jQuery` that only handles animations. I can't comment on the former since I'm atrociously awful at `CSS` animation, but I have used [Velocity.js](https://github.com/julianshapiro/velocity) as a replacement for `jQuery` before. It only contains animation libraries and should be smaller than `jQuery`, while also having more responsive animations in comparison.

Reply:
I think vuetify also has transitions that could be helpful here